src/test/GameyeMock.php

```diff
@@ -18,16 +18,16 @@ public function mockGame()
         "css": { "gameKey": "css", "location": {} },
         "l4d2": { "gameKey": "l4d2", "location": {} },
         "kf2": { "gameKey": "kf2", "location": {} },
-        "test": { "gameKey": "test", "location": { "100": true } }
+        "test": { "gameKey": "test", "location": { "local": true } }
     },
     "location": {
-        "1": { "locationKey": 1, "locationName": "Rotterdam" },
-        "2": { "locationKey": 2, "locationName": "Ireland" },
-        "3": { "locationKey": 3, "locationName": "Dubai" },
-        "4": { "locationKey": 4, "locationName": "Tokyo" },
-        "5": { "locationKey": 5, "locationName": "Los Angeles" },
-        "6": { "locationKey": 6, "locationName": "Washington DC" },
-        "100": { "locationKey": 100, "locationName": "Local" }
+        "rotterdam": { "locationKey": "rotterdam" },
+        "ireland": { "locationKey": "ireland" },
+        "dubai": { "locationKey": "dubai" },
+        "tokyo": { "locationKey": "tokyo" },
+        "los_angeles": { "locationKey": "los_angeles" },
```

Comment:
Do we solve white spaces with an underscore?

Reply:
Yes! we do!

i have a database patch that lowercases all names and replaces weird characters (like spaces) with underscores.